mmdet/datasets/pipelines/transforms.py

```diff
@@ -1903,3 +1905,195 @@ def __repr__(self):
                      else f'cutout_shape={self.candidates}, ')
         repr_str += f'fill_in={self.fill_in})'
         return repr_str
+
+
+@PIPELINES.register_module()
+class RandomAffine:
+    """Random affine transform data augmentation.
+
+    This operation randomly generates affine transform matrix which including
+    rotation, translation, shear and scaling transforms.
+
+    Args:
+        rotate_degree (float): Maximum degrees of rotation transform.
+            Default: 10.
+        translate_ratio (float): Maximum ratio of translation. Default: 0.1.
+        scaling_ratio (tuple[float]): Min and max ratio of scaling transform.
+            Default: (0.5, 1.5).
+        shear_degree (float): Maximum degrees of shear transform. Default: 2.
+        border (tuple[int]): Distance from height and width sides of input
+            image to adjust output shape. Only used in mosaic dataset.
+            Default: (0, 0).
+        border_val (tuple[int]): Border padding values of 3 channels.
+            Default: (114, 114, 114).
+        wh_filter_thr (float): Width and height threshold to filter bboxes.
```

Comment:
Seem we always use `min` and `max` as prefix when representing the threshold of the shape of bbox and image, such as `max_shape` in bboxcoder and `min_bbox_size` in RPN, seem `thr` only used when it is related to score